### PR TITLE
Sprint 7 task3

### PR DESCRIPTION
**Pull request by buzlan**
Implement some logic for Confirmation page: 
1) if i click "Далее" from orderDelivery screen i will see Confirmation page with some info: 
Screenshot_1637306608
Screenshot_1637306665
Screenshot_1637306669
Screenshot_1637306671
2) if i cahnge paid type i will see some changed info: 
Screenshot_1637306711
3) for orderTakeaway and reserveTable i have same logic.

### CODE REVIEW

File: src/Screens/ConfirmationScreen/confirmationPage.tsx [hunk 1]
```diff
@@ -1,10 +1,233 @@
...
+                </View>
+                {excludedIngredients[index]?.length > 0 ? (
+                  <View style={confirmationPageStyles.excludedIngWrapper}>
+                    <Text style={confirmationPageStyles.withoutIngText}>
+                      Без добавления:
+                    </Text>
+                    <Text style={confirmationPageStyles.ingTextWrapper}>
+                      {excludedIngredients[index]?.join(", ")}
+                    </Text>
+                  </View>
```
Comment (fedarenchyk) [inline src/Screens/ConfirmationScreen/confirmationPage.tsx:76]: move to the common component and reuse after

File: src/Screens/ConfirmationScreen/confirmationPage.tsx [hunk 2]
```diff
@@ -1,10 +1,233 @@
...
+                ) : null}
+                <View style={confirmationPageStyles.priceAndIconsContainer}>
+                  <Text style={confirmationPageStyles.priceItem}>
+                    {item.dish.price} BYN
+                  </Text>
+                  <Text style={confirmationPageStyles.priceItem2}>
+                    за 1 позицию
+                  </Text>
+                </View>
```
Comment (fedarenchyk) [inline src/Screens/ConfirmationScreen/confirmationPage.tsx:85]: move to the common component and reuse after

File: src/Screens/ConfirmationScreen/confirmationPage.tsx [hunk 3]
```diff
@@ -1,10 +1,233 @@
...
+          <View style={confirmationPageStyles.lineView}></View>
+          <View style={confirmationPageStyles.itemsWrapperContainer}>
+            <View style={confirmationPageStyles.itemWrapperStyle}>
+              <Text style={confirmationPageStyles.firstPartTextStyle}>
+                Тип заказа:
+              </Text>
+              <Text style={confirmationPageStyles.secondPartTextStyle}>
+                {orderData.orderType}
+              </Text>
+            </View>
```
Comment (fedarenchyk) [inline src/Screens/ConfirmationScreen/confirmationPage.tsx:98]: move to the common component and reuse after

File: src/Screens/ConfirmationScreen/confirmationPage.tsx [hunk 4]
```diff
@@ -1,10 +1,233 @@
...
+                <Text style={confirmationPageStyles.firstPartTextStyle}>
+                  Адрес:
+                </Text>
+                <Text
+                  style={confirmationPageStyles.secondPartTextStyle}
+                  key={uuid.v4()}
+                >
+                  {formatAdress(orderData.adress)}
+                </Text>
+              </View>
```
Comment (fedarenchyk) [inline src/Screens/ConfirmationScreen/confirmationPage.tsx:110]: move to the common component and reuse after

File: src/Screens/ConfirmationScreen/confirmationPage.tsx [hunk 5]
```diff
@@ -1,10 +1,233 @@
...
+            ) : null}
+            <View style={confirmationPageStyles.itemWrapperStyle}>
+              <Text style={confirmationPageStyles.firstPartTextStyle}>
+                Дата:
+              </Text>
+              <Text style={confirmationPageStyles.secondPartTextStyle}>
+                {orderData.date}
+              </Text>
+            </View>
```
Comment (fedarenchyk) [inline src/Screens/ConfirmationScreen/confirmationPage.tsx:119]: move to the common component and reuse after

File: src/Screens/ConfirmationScreen/confirmationPage.tsx [hunk 6]
```diff
@@ -1,10 +1,233 @@
...
+            <View style={confirmationPageStyles.itemWrapperStyle}>
+              <Text style={confirmationPageStyles.firstPartTextStyle}>
+                Время:
+              </Text>
+              <Text style={confirmationPageStyles.secondPartTextStyle}>
+                {orderData.time}
+              </Text>
+            </View>
```
Comment (fedarenchyk) [inline src/Screens/ConfirmationScreen/confirmationPage.tsx:127]: move to the common component and reuse after

File: src/Screens/ConfirmationScreen/confirmationPage.tsx [hunk 7]
```diff
@@ -1,10 +1,233 @@
...
+            <View style={confirmationPageStyles.contactInformationStyleWrapper}>
+              <Text style={confirmationPageStyles.firstPartTextContactStyle}>
+                Контактная информация:
+              </Text>
+
+              <Text style={confirmationPageStyles.secondPartTextContactStyle}>
+                Кристина, +375296457721
+              </Text>
+            </View>
```
Comment (fedarenchyk) [inline src/Screens/ConfirmationScreen/confirmationPage.tsx:136]: move to the common component and reuse after

File: src/Screens/ConfirmationScreen/confirmationPage.tsx [hunk 8]
```diff
@@ -1,10 +1,233 @@
...
+            {orderData.table ? (
+              <View style={confirmationPageStyles.itemWrapperStyle}>
+                <Text style={confirmationPageStyles.firstPartTextStyle}>
+                  Стол на
+                </Text>
+                <Text style={confirmationPageStyles.secondPartTextStyle}>
+                  {orderData.table}
+                </Text>
+              </View>
```
Comment (fedarenchyk) [inline src/Screens/ConfirmationScreen/confirmationPage.tsx:145]: move to the common component and reuse after

File: src/Screens/ConfirmationScreen/confirmationPage.tsx [hunk 9]
```diff
@@ -1,10 +1,233 @@
...
+          <View style={confirmationPageStyles.lineView}></View>
+          <View style={{ paddingTop: 30, alignItems: "center" }}>
+            <View style={confirmationPageStyles.itemWrapperStyle}>
+              <Text style={confirmationPageStyles.firstPartTextStyleResult}>
+                Итого:
+              </Text>
+              <Text style={confirmationPageStyles.secondPartTextStyleResult}>
+                {totalSum} BYN
+              </Text>
+            </View>
```
Comment (fedarenchyk) [inline src/Screens/ConfirmationScreen/confirmationPage.tsx:157]: move to the common component and reuse after

File: src/Screens/ConfirmationScreen/confirmationPage.tsx (hunk 1 above)
Comment (buzlan) [reply src/Screens/ConfirmationScreen/confirmationPage.tsx:76]: fixed

File: src/Screens/ConfirmationScreen/confirmationPage.tsx (hunk 2 above)
Comment (buzlan) [reply src/Screens/ConfirmationScreen/confirmationPage.tsx:85]: fixed

File: src/Screens/ConfirmationScreen/confirmationPage.tsx (hunk 3 above)
Comment (buzlan) [reply src/Screens/ConfirmationScreen/confirmationPage.tsx:98]: fixed

File: src/Screens/ConfirmationScreen/confirmationPage.tsx (hunk 4 above)
Comment (buzlan) [reply src/Screens/ConfirmationScreen/confirmationPage.tsx:110]: fixed

File: src/Screens/ConfirmationScreen/confirmationPage.tsx (hunk 5 above)
Comment (buzlan) [reply src/Screens/ConfirmationScreen/confirmationPage.tsx:119]: fixed

File: src/Screens/ConfirmationScreen/confirmationPage.tsx (hunk 6 above)
Comment (buzlan) [reply src/Screens/ConfirmationScreen/confirmationPage.tsx:127]: fixed

File: src/Screens/ConfirmationScreen/confirmationPage.tsx (hunk 7 above)
Comment (buzlan) [reply src/Screens/ConfirmationScreen/confirmationPage.tsx:136]: fixed

File: src/Screens/ConfirmationScreen/confirmationPage.tsx (hunk 8 above)
Comment (buzlan) [reply src/Screens/ConfirmationScreen/confirmationPage.tsx:145]: fixed

File: src/Screens/ConfirmationScreen/confirmationPage.tsx (hunk 9 above)
Comment (buzlan) [reply src/Screens/ConfirmationScreen/confirmationPage.tsx:157]: fixed

**Review by fedarenchyk — APPROVED** [review]

**Pull request merged**